src/HDInsight/HDInsight/ManagementCommands/NewAzureHDInsightClusterCommand.cs

```diff
@@ -717,7 +718,7 @@ private Guid GetApplicationId(Guid applicationId)
                 string errorMessage =$"Can not find service princaipl per the parameter ObjectId:{ObjectId}, the error message is '{e.Message}'."+ " Please specify Application Id explicitly by providing ApplicationId parameter and retry.";
                 throw new AzPSArgumentException(errorMessage, nameof(ObjectId));
             }
-
+            
```

Comment:
Remove space here make sure the good code style, pls check all files remove duplicate empty lines and extra spaces